config/migrations/2024/20241014141246-Decisions-Types.ttl

```diff
@@ -869,10 +869,11 @@ BesluitType:fb92601a-d189-4482-9922-ab0efc6bc935
 BesluitType:ec8652a6-2c49-4955-8d59-fd08c8c0a6e1
     a skos:Concept ;
     core:uuid "ec8652a6-2c49-4955-8d59-fd08c8c0a6e1" ;
-    besluit:notificationRule lblodRule:e88a5f18-7c9d-456c-8bfa-e3323bbb29f3 ;
```

Comment:
We still want to keep the link to the notification rule no?

Reply:
I think it's not needed anymore since we use `besluit:decidableBy BestuurseenheidClassificatieCode:36a82ba0-7ff1-4697-a9dd-2e94df73b721 ; # Autonoom gemeentebedrijf` now
This besluittype was added 2 years ago, but never had any form. But I could be wrong since it's my first time working in this context.

Reply:
I also don't have a lot of context on that, might be worth asking someone working closer to those forms? In any case, if we keep it then you'll have to restore the link and if we don't keep it you'll have to remove the notification rule from the ttl as well as in a migration.

Reply:
Got it, I asked @Riadabd before and he agreed it could be deleted

Reply:
In case of anyone hitting this discussion: These notification rules are used by vendors to see who can send what and other info. So best to keep them. See also https://lblod.github.io/pages-vendors/#/docs/meldingsplicht